src/components/App.js

```diff
@@ -47,20 +48,20 @@ class App extends React.Component {
 
   render() {
     return (
-      <div className="section__container">
-        <div className="section__container__a">
-        <PreviewCard />
+      <div className='section__container'>
+        <div className='section__container__a'>
+          <Landing />
```

Comment:
The landing component has to be on top of all, and not inside section__container__a.